src/OmniSharp.Roslyn.CSharp/Helpers/DiagnosticExtensions.cs

```diff
@@ -18,47 +18,27 @@ internal static DiagnosticLocation ToDiagnosticLocation(this Diagnostic diagnost
                 Column = span.StartLinePosition.Character,
```

Comment:
how they deal with mapped location and etc? and additional location from diagnostic id?

Reply:
missed the GetMappedLineSpan call at the top. in that case. this line column might not match the text user is seeing at the editor. is that okay?